src/Data/Ratio.purs

```diff
@@ -8,11 +8,15 @@ module Data.Ratio
 import Prelude ( class CommutativeRing, class Eq, class EuclideanRing
                , class Field, class Ring, class Semiring
                , one, zero, (*), (+), (-)
+               , class Show, show, (<>)
                )
 import Prelude ( gcd ) as Prelude
 
 data Ratio a = Ratio a a
 
+instance showRatio :: Show a => Show (Ratio a) where
+  show (Ratio a b) = "Ratio " <> show a <> " " <> show b
```

Comment:
Could you add parenthesis around this whole expression? It will be then safer to copy&paste this output into other expression.